### PR TITLE
Added Kensei

### PR DESCRIPTION
**Pull request by JohnathanShiao**

### CODE REVIEW

**Review by TresTres — CHANGES_REQUESTED** [review]
please account for the weapon proficiency

**Comment by JohnathanShiao** [comment]
fixed weapon proficiencies

File: src/Classes/Monk/Subclasses/Kensei/Kensei.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      }
+    }, 
+    "17": {
+      "UNERRING ACCURACY": {
+        "title": "Unerring Accuracy", 
+        "description": "At 17th level, your mastery of weapons grants you extraordinary accuracy. If you miss with an attack roll using a monk weapon on your turn, you can reroll it. You can use this feature only once on each of your turns."
+      }
+    }
+  }
+<<<<<<< HEAD
```
Comment (TresTres) [inline src/Classes/Monk/Subclasses/Kensei/Kensei.json:30]: You have some artifacts like this from merging that shouldn't be here

File: src/Classes/Monk/Subclasses/MonkSubclass.ts [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import { Subclass } from "../../Subclass";
+<<<<<<< HEAD
```
Comment (TresTres) [inline src/Classes/Monk/Subclasses/MonkSubclass.ts:2]: in this file too

**Review by TresTres — CHANGES_REQUESTED** [review]

**Review by alimohamad — APPROVED** [review]
10/10

**Pull request merged**